common/test/acceptance/pages/studio/overview.py

```diff
@@ -1037,19 +1027,48 @@ def policy(self, grading_label):
         ).fulfill()
 
     @property
+    def is_staff_lock_visible(self):
+        """
+        Returns true if the staff lock option is visible, either as a checkbox
+        (section and unit levels) or a radio button (subsection level).
+        """
+        return self.find_css('#staff_lock').visible or self.find_css('input[name=content-visibility]').visible
+
+    def ensure_staff_lock_visible(self):
+        """
+        Ensures the staff lock option is visible, clicking on the advanced tab
+        if needed.
+        """
+        if not self.is_staff_lock_visible:
+            self.find_css(".settings-tab-button[data-tab=advanced]").click()
```

Comment:
You're right - I'm not a fan of the subsection/nonsubsection logic in these tests. I'll clean that up.